performance/FireperfGames.tulsiproj/Configs/all.tulsigen

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
This seems g3-specific? What's it for?

Reply:
I don't know. This is just what was pulled from the export script. You're right though, looks like we can drop it.

Reply:
Looks like this can be removed.